blog/keycloak-newsletter-2.ftl

```diff
@@ -0,0 +1,81 @@
+<#--
+title = Keycloak Community Newsletter #2
+date = 2019-05-02
+publish = false
+author = Sébastien Blanc
+-->
+
+<p>
+A lot of news to share in this second edition, so fasten your seatbelt and let's go ! 
+</p>
+
+
+<h2>News from the community</h2>
+<p>
+First of all, we would like to thank the whole community that has contributed to this edition by sharing their links, tips, etc ...
+</p>
+<p>
+More and more API Management/Gateway solutions can now be integrated with Keycloak. Recently, <a href="https://www.getambassador.io/">Ambassador</a>, a Open Source Kubernetes-Native API Gateway built on the Envoy Proxy, has published a <a href="https://www.getambassador.io/tutorials/github-sso/">quickstart</a> that shows the Keycloak's integration.   
```

Comment:
"that shows the Keycloak's integration" is incorrect English, the post also focuses on GitHub, not Keycloak integration?